CMakeLists.txt

```diff
@@ -157,6 +157,8 @@ option(QUDA_COVDEV "build code for covariant derivative" OFF)
 option(QUDA_LAPLACE "build laplace operator" OFF)
 option(QUDA_QIO "build QIO code for binary I/O" OFF)
 
+option(QUDA_TWOLINK_GSMEAR "build code for two-link Gaussian smearing" OFF)
```

Comment:
This option should have a name that will allow us to aggregate similar options together in the `ccmake` list.  E.g., `QUDA_SMEAR_GAUSS_TWOLINK` which would mean that if we have other `QUDA_SMEAR_....` options they will appear next to each other in the ccmake list.

Reply:
done